website/cue/reference/remap.cue

```diff
@@ -83,6 +83,7 @@ remap: #Remap & {
 			}
 			warnings: [
 				"All attributes are strings and will require manual type coercing.",
+				"Values of duplicate keys are grouped into an array.",
```

Comment:
I think you meant to add this to `website/cue/reference/remap/functions/parse_key_value.cue` ? The warning here just applies to that function, not `remap` generally.

We should also document this new behavior there and probably add a new example.

Reply:
Done.